### PR TITLE
Introduce alembic to PPR API

### PR DESCRIPTION
**Pull request by jguertin**
Introduce alembic to PPR API in order to support addition of search table

New files in this commit were generated by running `alembic init migrations`

### CODE REVIEW

**Review by WalterMoar — APPROVED** [review]
Great? I'm not familiar with Alembic so it's all magic to me.

File: ppr-api/migrations/README [hunk 1]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (WalterMoar) [inline ppr-api/migrations/README:1]: GitHub seems really picky about wanting a newline on the last line.

**Comment by bryan-gilbert** [comment]
> Great? I'm not familiar with Alembic so it's all magic to me.

ditto for me

File: ppr-api/migrations/README (hunk 1 above)
Comment (jguertin) [reply ppr-api/migrations/README:1]: Will correct with the next PR.  I'm going to add some alembic usage notes with the first migration.

**Pull request merged**